frontends/web/src/locales/en/app.json

```diff
@@ -1184,12 +1184,7 @@
   },
   "settings": {
     "accounts": {
-      "bitcoinP2PKH": "Bitcoin Legacy",
-      "bitcoinP2WPKH": "Bitcoin: bech32",
-      "bitcoinP2WPKHP2SH": "Bitcoin",
       "ethereum": "Ethereum",
-      "litecoinP2WPKH": "Litecoin: bech32",
-      "litecoinP2WPKHP2SH": "Litecoin",
```

Comment:
Shouldn't this be removed from all the other translations as well?

Reply:
If I remember correctly, removing needs to be done only in the reference lang (en), then locize-push will delete it for the other langs on locize.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/docs/i18n.md seems to confirm. I'll try.

Reply:
Confirmed, next locize-pull will remove them.

Reply:
thanks!